### DISCUSSION TITLE
Color code click markers of the replay analysis

### DISCUSSION THREAD

**Discussion by TechniKris** [ppy/osu#29827] [Ideas] [answered]
Key 1 and key 2 click markers are difficult to distinguish while a replay is playing.
I suggest coloring them with distinct colors.
Perhaps frame markers could be colored in a similar way.
Direct comparison with Rewind:
![obraz](https://github.com/user-attachments/assets/5c4f032f-661e-4b84-95a1-9d77b20f9429)

On a sidenote: I think Rewind's UX is very good and copying a lot of its design would be beneficial.

**Comment by peppy** [comment, marked as answer]
We intentionally *didn't* colour them because it's hard to discern and a lot of people just confused them as the hit result colours.

If you're going to compare to another display, please at least make the display the same (you've hidden the line on one, and it's using a different skin).

Also to me, the click types are more easily visible in the lazer display.
